### PR TITLE
Add version collection

### PR DESCRIPTION
**Pull request by ritarora**
Add version Collection for  https://github.wdf.sap.corp/gd/FioriExperience/issues/196

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -183,4 +183,18 @@ collections:
...
+    folder: "src/pages/versions"
+    create: false
+    extension: "json"
+    format: "json"
+    editor:
+      preview: false
+    slug: "{{slug}}"
+    identifier_field: srcTemplateKey
+    fields:
+    - {label: "Template Key", name: "templateKey", widget: "hidden", default: version-config"}
```
Comment (devinea) [inline static/admin/config.yml]: missing `"` on default value. 
Probably the cause of the extra `\"` in the json files

Comment (ritarora) [reply static/admin/config.yml]: oh, will update, thanks

File: static/admin/config.yml [hunk 2]
```diff
@@ -183,4 +183,18 @@ collections:
               - label: "Color Description"
                 name: "description"
                 widget: "string"
-      - {label: "Version No.", name: "version", widget: "select", options: ["1.01", "1.02", "1.03","1.04", "1.05"]}
\ No newline at end of file
+      - {label: "Version No.", name: "version", widget: "select", options: ["1.01", "1.02", "1.03","1.04", "1.05"]}
+  - name: "versions"
+    label: "Versions"
```
Comment (devinea) [inline static/admin/config.yml]: Maybe label should be `Set Guideline Current Version`

**Comment by devinea** [comment]
Thanks @ritarora

**Pull request merged**